compiler/utils.ml

```diff
@@ -18,4 +18,4 @@ let str_of_file file =
     | Some(s) -> aux (s :: acc) in
   aux []
 
-let conclude_program () = "if PLOT:\n\tplt.show()"
+let conclude_program () = "if PLOT:\n\timport matplotlib.pyplot as plt\n\tplt.show()"
```

Comment:
is it necessary to import again here? won't plt.show() only be called if the function has been run and plt already imported? or is plt local to that function?